packages/backend/src/panels/YeomanUIPanel.ts

```diff
@@ -47,6 +47,7 @@ export class YeomanUIPanel extends AbstractWebviewPanel {
       }
     }
 
+    this.setViewColumn(uiOptions && uiOptions.viewColumn ? uiOptions.viewColumn : vscode.ViewColumn.One);
```

Comment:
there are 2 webviews in yeoman-ui and all of them should by default get viewColumn.One
Just place this logic loadWebviewPanel of the AbstractWebviewPanel.ts

You do not need a new method just do the following:
this.viewColumn = uiOptions?.viewColumn ?? vscode.ViewColumn.One;

In addition check that uiOptions?.viewColumn has valid value according to vscode.VewColumn enum

Please add unit test

Reply:
@slavik-lvovsky : Thanks for the review. I've made the suggested changes.